### ISSUE TITLE
S3DIS from the scrathch

### ISSUE THREAD

**Issue by ImaneTopo** [oneformer3d/oneformer3d#60]
Hello, I want to train the model on the s3dis dataset from the scrath, is ti possible and how?

**Comment by oneformer3d-contributor** [comment]
Does #54 help?

**Comment by ImaneTopo** [comment]
I cannot work with the pretrained on the scannet, because I want on the s3dis from the scratch in order to work with it in another task because my original dataset format  is similar to the s3dis ( X Y Z RGB).

**Comment by ImaneTopo** [comment]
> Does #54 help?

I cannot work with the pretrained on the scannet, because I want to train the model on the s3dis from the scratch in order to work with it in another task because my original dataset format  is similar to the s3dis ( X Y Z RGB).

**Comment by oneformer3d-contributor** [comment]
I think, all of our datasets are (x, y, z, rgb). So why not to start from pre-trained checkpoint? If you don't want to do it, just delete the `load_from` parameter from config file.

**Comment by ImaneTopo** [comment]
> I think, all of our datasets are (x, y, z, rgb). So why not to start from pre-trained checkpoint? If you don't want to do it, just delete the `load_from` parameter from config file.

My task for the segmentation is related to agriculture (specifically fruits count) so I'm not working on indoor, for that reason I choose to train the model from the scratch with my own data prepared that it's format is simalar to S3DIS.

**Comment by ImaneTopo** [comment]
> > I think, all of our datasets are (x, y, z, rgb). So why not to start from pre-trained checkpoint? If you don't want to do it, just delete the `load_from` parameter from config file.
> 
> My task for the segmentation is related to agriculture (specifically fruits count) so I'm not working on indoor, for that reason I choose to train the model from the scratch with my own data prepared that it's format is simalar to S3DIS.

* my task is for instance segmentation of fruits on trees.

**Comment by oneformer3d-contributor** [comment]
> just delete the `load_from` parameter from config file

Does this answers your question?

**Comment by ImaneTopo** [comment]
> > just delete the `load_from` parameter from config file
> 
> Does this answers your question?

Thank you  for your response. So this change will not influence the precisions?

**Comment by oneformer3d-contributor** [comment]
Hard to say anything about your dataset, as we only tried for indoor environments. On S3DIS this pre-training brings quite some improvement in accuracy, because the dataset is very small.
